content/hi/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,50 @@
+---
+title: कंटेनर रनटाइम इंटरफेस
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+CRI एक प्लगइन इंटरफ़ेस है जो क्यूबलेट को क्लस्टर घटकों को पुन: संकलित करने की आवश्यकता के बिना, विभिन्न प्रकार के कंटेनर रनटाइम का उपयोग करने में सक्षम बनाता है।
+
+आपको काम करने की जरूरत है
+{{<glossary_tooltip text="कंटेनर रनटाइम" term_id="container-runtime">}} चालू
+आपके क्लस्टर में प्रत्येक नोड, ताकि
+{{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} लॉन्च हो सकता है
+{{< glossary_tooltip text="पॉड्स" term_id="pod" >}} और उनके कंटेनर।
+
+{{< glossary_definition prepend="The Container Runtime Interface (CRI) is" term_id="container-runtime-interface" length="all" >}}
+
+<!-- body -->
+
+## एपीआई {#api}
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+जीआरपीसी के माध्यम से कंटेनर रनटाइम से कनेक्ट होने पर क्यूबलेट क्लाइंट के रूप में कार्य करता है।
+कंटेनर में रनटाइम और इमेज सर्विस एंडपॉइंट उपलब्ध होने चाहिए
+रनटाइम, जिसे उपयोग करके क्यूबलेट के भीतर अलग से कॉन्फ़िगर किया जा सकता है
+`--image-service-endpoint` और `--container-runtime-endpoint` [कमांड लाइन
+फ़्लैग](/दस्तावेज़/संदर्भ/कमांड-लाइन-उपकरण-संदर्भ/क्यूबलेट)
+
+कुबेरनेट्स v{{< skew currentVersion >}} के लिए, क्यूबलेट CRI `v1` का उपयोग करना पसंद करता है।
+यदि एक कंटेनर रनटाइम CRI के `v1` का समर्थन नहीं करता है, तो क्यूबलेट कोशिश करता है
+किसी पुराने समर्थित संस्करण पर बातचीत करें।
+v{{< skew currentVersion >}} क्यूबलेट CRI `v1alpha2` पर भी बातचीत कर सकता है, लेकिन
+इस संस्करण को पदावनत माना जाता है।
+यदि क्यूबलेट समर्थित सीआरआई संस्करण पर बातचीत नहीं कर सकता है, तो क्यूबलेट हार मान लेता है
+और नोड के रूप में पंजीकृत नहीं होता है।
+
+## उन्नयन
```

Comment:
Consider incorporating English text for the link to avoid non-English characters in the URL of the section.

```suggestion
## उन्नयन {#upgrading}
```